Expecto/Model.fs

```diff
@@ -88,7 +88,7 @@ type Test =
   /// A collection/list of tests.
   | TestList of tests:Test list * state:FocusState
   /// A labelling of a Test (list or test code).
-  | TestLabel of label:string * test:Test * state:FocusState
+  | TestLabel of label:string list * test:Test * state:FocusState
```

Comment:
Wouldn't it be possible to have a single label here? Because it's only the composition of these `Test` values that is the array; i.e. FlatTest would have an array, but a label shouldn't have to be an array

Reply:
So simply keep here a name only relevant for a particular test (`test`), and when we log/get names then traverse the whole tree and build names like `groupA/subgroupAA/testA` or `groupA.subgroupA.testA`, right?

Reply:
@haf upup :)?

Reply:
Yes exactly:)

Reply:
Yes exactly:)

Reply:
Done